lib/vtls/vtls.c

```diff
@@ -245,6 +245,7 @@ Curl_ssl_connect_nonblocking(struct connectdata *conn, int sockindex,
                              bool *done)
 {
   CURLcode result;
+  infof(conn->data, "Using ssl backend: %s\n", Curl_ssl->info.name);
```

Comment:
Here this is called multiple times IIRC so you could end up with
Using ssl backend: openssl
Using ssl backend: openssl
Using ssl backend: openssl

Reply:
Ah right, I suspected that was the case... Thanks.